bk-api/bk_api.py

```diff
@@ -1316,102 +1253,95 @@ def get(self):
         Returns:
             dict: end-point id, private key, public key and certificate
         """
-        node_id = request.args.get("id", type=str)
+        logger.info("registration request: %r", request.args)
+
+        node_id = request.args.get("node_id", type=str)
```

Comment:
i noticed you changed this from `id` to `node_id`... just thinking through this...

since this is called from the `register.sh` and no-where else this interface argument name change is okay (right?).

Reply:
Yes, I believe the only user of this /register API directly is that script and I updated both to match.